plugins/out_influxdb/influxdb.c

```diff
@@ -665,6 +690,12 @@ static struct flb_config_map config_map[] = {
      "Space separated list of keys that needs to be tagged."
     },
 
+    {
+     FLB_CONFIG_MAP_BOOL, "use_integer", "false",
```

Comment:
maybe call it `add_integer_suffix`

Reply:
Thanks for the suggestion. Fixed in https://github.com/fluent/fluent-bit/pull/9301/commits/6f6bfcbc39e6e75713d1a0c81e8d929ce9e17ad3